Makefile

```diff
@@ -147,26 +147,25 @@ integrationtest:
 	edge/test/integration/scripts/execute.sh
 endif
 
-CROSSBUILD_COMPONENTS=edgecore
 GOARM_VALUES=GOARM7 \
 	GOARM8
 
 define CROSSBUILD_HELP_INFO
 # cross build components.
 #
 # Args:
-#   WHAT: Component names to be lint check. support: $(CROSSBUILD_COMPONENTS)
+#   WHAT: Component names to be lint check. support: $(BINARIES)
 #         If not specified, "everything" will be cross build.
 #
-# GOARM: go arm value, now support:$(GOARM_VALUES)
-#        If not specified ,default use GOARM=GOARM8
+# GOARM: arm version for go crossbuild, now support:$(GOARM_VALUES)
+#        If not specified, default use ARM=GOARM8
```

Comment:
Hmm... GOARM is the original args, but you have updated this to "ARM",  so it would be something like,

```
ARM: arm version....
```

But I still think using `ARM` here is a little misleading.

Reply:
Yes, I also think `ARM` maybe misleading.. ;-)